tox.ini

```diff
@@ -312,46 +390,101 @@ deps =
 basepython = python2.7
 
 commands =
-    "{toxinidir}/.travis/environment"
+    "{toxinidir}/.travis/twistedchecker-diff" {posargs:{env:PY_MODULE}}
 
-    "{toxinidir}/.travis/twistedchecker-diff" {posargs:klein}
+
+##
+# Coverage report
+##
+
+[testenv:coverage_report]
+
+description = generate coverage report
+
+depends = {test,coverage}-py{27,35,36,37,38,py2,py3}-tw{171,184,current,trunk}
```

Comment:
Wait for test environments to complete before running this one (only matters if you run them in parallel).